### ISSUE TITLE
Unable to navigate with arrow keys to the bottom records of navigatable Grid with virtual scrolling and local datasource

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-ui-core#8156]
### Bug report
**Regression since 2022.1.119** . The last version navigating without issue is 2021.3.1207 - [Dojo](https://dojo.telerik.com/VlGXhGyG)

Note: if the grid uses remote data, the issue is not reproducible -  when more data is needed to be loaded, a loader is shown and after the data is loaded, scrolling down continues as expected - [Dojo](https://dojo.telerik.com/pVAyuZxW)

### Reproduction of the problem
1. Run the following [Dojo](https://dojo.telerik.com/BjuCqkiY)
2. Click on the first record of the Grid - cell 1 in ID column - the cell is focused
3. Press and hold Down arrow key

### Current behavior
The Grid starts scrolling down and focusing other cells. At some point the Grid stops scrolling down - it may happen on reaching different records, sometimes even on record 600+, sometimes on 46.
If we scroll the Grid, navigating downwards may continue.

### Expected/desired behavior
The Grid starts scrolling down and focusing other cells until the last cell is reached and focused.
The last version navigating without issue is 2021.3.1207 - [Dojo](https://dojo.telerik.com/VlGXhGyG)

### Environment
* **Kendo UI version:** 2025.1.211
* **jQuery version:** x.y
* **Browser:** all

**Comment by iordanGrancharov** [comment]
This issue is not a regression, but rather the result of a misconfiguration.

The Grid's virtualization feature has certain known limitations that cannot be bypassed. In all provided examples, the following guideline has not been observed:

 `Provide for a page size of the Grid that is large enough, so that the table rows do not fit in the scrollable data area. Otherwise the vertical virtual scrollbar will not be created. The page size of the Grid must be over three times larger than the number of visible table rows in the data area.` 

When the `pageSize` is set to at least three times the number of visible rows, keyboard navigation functions as expected.

The difference in behavior between versions is due to the transition from `em` to `rem` units in the component's styles.

- `em` is relative to the font size of the parent element, while `rem` is relative to the font size of the root (`html`) element

As a result, the `font-size` applied to the `html` tag via the style attribute also affects the `padding` of the rows leading, to significant miscalculations.
